packages/components/src/controllers/useSizeOverride.browser.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import { describe, it, expect, beforeEach } from "vitest";
+import { mount } from "@arcgis/lumina-compiler/testing";
+import { h, JsxNode, LitElement } from "@arcgis/lumina";
+import { createRef } from "lit/directives/ref.js";
+import { useSizeOverride } from "./useSizeOverride";
+
+describe("useSizeOverride", async () => {
+  class Test extends LitElement {
+    ref = createRef<HTMLDivElement>();
+    resizeValues = { inlineSize: null, blockSize: null };
+
+    controller = useSizeOverride({
+      targetElement: () => this.ref.value,
+      getBounds: (axis) => (axis === "inline" ? { min: 100, max: 500 } : { min: 60, max: 400 }),
+    });
+
+    applySizeOverride(sizes: { inline?: number | null; block?: number | null }): void {
```

Comment:
I think you can simplify by dropping `resizeValues`, `applySizeOverride` and asserting on the component's controller directly in tests:

```tsx
it("clamps size above max", () => {
    const size = component.controller.resize({ inline: 600 });
    expect(el.ref.value!.style.inlineSize).toBe("500px");
    expect(size.inline).toBe(500);
  });
```